DAO.sol

```diff
@@ -84,7 +84,7 @@ contract DAOInterface {
     uint public proposalDeposit;
 
     // the accumulated sum of all current proposal deposits
-    uint sumOfProposalDeposits;
+    uint public sumOfProposalDeposits;
```

Comment:
On purpose. Currently it's hard for users to know what is the actual balance. Either we make `actualBalance` non-internal, or this one public. Same result.